python/sanlock.c

```diff
@@ -201,39 +222,39 @@ __hosts_to_list(struct sanlk_host *hss, int hss_count)
             goto exit_fail;
 
         /* fill the dictionary information: host_id */
-        if ((ls_value = PyInt_FromLong(hss[i].host_id)) == NULL)
+        if ((ls_value = PyLong_FromLong(hss[i].host_id)) == NULL)
```

Comment:
From src/sanlock.h:
```
163 struct sanlk_host {                                                                                                                                                      
164         uint64_t host_id;
165         uint64_t generation;
166         uint64_t timestamp;
167         uint32_t io_timeout;
168         uint32_t flags;
169 };
```
host_id should be converted using:

    PyLong_FromUnsignedLongLong

In practice host_id is never more than 2000, but we want to match the sanlock types.
This issue exist also for rest of the fields.